src/Entity/Coalition/CauseFollower.php

```diff
@@ -2,20 +2,31 @@
 
 namespace App\Entity\Coalition;
 
+use ApiPlatform\Core\Annotation\ApiResource;
 use App\Entity\AbstractFollower;
 use App\Entity\Adherent;
 use App\Entity\FollowedInterface;
+use App\Entity\Geo\Zone;
+use App\Validator\Coalition\CauseFollowerEmail as AssertCauseFollowerEmailValid;
 use Doctrine\ORM\Mapping as ORM;
 use Ramsey\Uuid\Uuid;
 use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
 
 /**
- * @ORM\Entity
+ * @ApiResource(
+ *     itemOperations={"get"},
```

Comment:
```suggestion
 *     itemOperations={},
```

Reply:
Without `"get"` creation of an adherent follow is failed.